lib/foreman_maintain.rb

```diff
@@ -68,9 +81,28 @@ def init_logger
       # argument to Logger.new will not work as expected
       filename = File.expand_path("#{config.log_dir}/foreman-maintain.log")
       @logger = Logger.new(filename, 10, 10_240_000).tap do |logger|
-        logger.level = config.log_level || Logger::DEBUG
+        logger.level = LOGGER_LEVEL_MAPPING[config.log_level] || Logger::DEBUG
         logger.datetime_format = '%Y-%m-%d %H:%M:%S%z '
       end
+      popup_msg_queue
+    end
+
+    def popup_msg_queue
```

Comment:
The method should perhaps be called `pickup_log_messages`, as `popup` seems more like the messages should popup somewhere in the user interface